lib/driver.js

```diff
@@ -256,15 +259,21 @@ class AndroidUiautomator2Driver extends BaseDriver {
       await this.initAUT();
     }
     //Adding AUT package name in the capabilities if package name not exist in caps
-    if (!this.caps.appPackage) {
-      this.caps.appPackage = appInfo.appPackage;
+    if (this.opts.appInfo && !this.caps.appPackage) {
```

Comment:
sounds good! thanks for your idea and we'll be happy to assist when you've got another PR ready to look at.